mysql-test/suite/rocksdb/t/bulk_load_errors.test

```diff
@@ -0,0 +1,47 @@
+--source include/have_rocksdb.inc
+--source include/not_crashrep.inc
+
+--let SEARCH_FILE=$MYSQLTEST_VARDIR/tmp/rocksdb.bulk_load_errors.err
+--let $restart_parameters=restart:--log-error=$SEARCH_FILE
+--replace_result $SEARCH_FILE LOG_FILE
+--source include/restart_mysqld.inc
+
+CREATE TABLE t1(pk INT, PRIMARY KEY(pk)) ENGINE=RocksDB;
+
+# Make sure we get an error with out of order keys during bulk load
+SET rocksdb_bulk_load=1;
+INSERT INTO t1 VALUES(10);
+INSERT INTO t1 VALUES(11);
+--error ER_KEYS_OUT_OF_ORDER
+INSERT INTO t1 VALUES(9);
+SET rocksdb_bulk_load=0;
+
+# Make sure only 10 and 11 got into the table
+SELECT * FROM t1;
+
+# Make sure we get an error with overlapping data
+SET rocksdb_bulk_load=1;
+INSERT INTO t1 VALUES(1);
+INSERT INTO t1 VALUES(2);
+INSERT INTO t1 VALUES(20);
+INSERT INTO t1 VALUES(21);
+
+# This last crashes the server (intentionally) because we can't return any
+# error information from a SET <variable>=<value>
+--source include/expect_crash.inc
+--error 2013 # CR_SERVER_LOST
+SET rocksdb_bulk_load=0;
+
+# Make sure the error exists in the .err log and then restart the server
+--let SEARCH_PATTERN=RocksDB: Error 207 finalizing last SST file while setting bulk loading variable
+--source include/search_pattern_in_file.inc
+
+--let SEARCH_PATTERN=
+--let SEARCH_FILE=
+--let $restart_parameters=
+--source include/start_mysqld.inc
+
+--enable_reconnect
+--source include/wait_until_connected_again.inc
+
+DROP TABLE t1;
```

Comment:
Doh, yup, you're right, and I should have caught this. I usually do --repeat=3 before pushing but obviously forgot this time. This specific sequence (use alternate .err log file) seems frequent enough that I wonder if there is some relatively easy way to automate this via some --source and validate proper cleanup. Maybe even going a little higher level and add some facility to mtr like 'get_tmp_filename' that gives a properly pathed tmp file name and validates that the test has cleaned up (or auto cleanup?) on exit.